envi/archs/thumb16/disasm.py

```diff
@@ -700,59 +718,62 @@ def ThumbExpandImm_C(imm4, imm8, carry):
             return imm8 | (imm8 << 8) | (imm8 << 16) | (imm8 << 24), carry
     else:
         a = imm8 >> 7
-        imm8 |= 0x80 # 1bcdefg
-        off = (a | (imm4<<1) )
+        imm8 |= 0x80  # 1bcdefg
+        off = (a | (imm4 << 1))
         return ROR_C(imm8, 32, off)
 
+
 dp_secondary = (
-            'tst',# and
-            None, # bic
-            'mov', # orr
-            'mvn', # orn
-            'teq', # eor
-            None, #
-            None, #
-            None, # pkh
-            'cmn', #add
-            None, # adc
-            None, # sbc
+    'tst',  # and
```

Comment:
I believe this is from autopep8. I'm not super happy with it, but I leaned fairly heavily on tooling like flake8 to massively speed up my search for old syntax. But just the sheer size of the file and various things flake8 nitpicked about made the output a nightmare to search through and I forgot to revert some of the spacing back.

I'll go back through and clean it up since autopep8 seems to be absolutely terrible when it comes to formatting. I haven't had issues like this in the past with it, so I'm wondering what happened.